examples/storage/cassandra/cassandra-petset.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: "apps/v1alpha1"
+kind: PetSet
+metadata:
+  name: cassandra
+spec:
+  serviceName: cassandra
+  replicas: 3
+  template:
+    metadata:
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+      labels:
+        app: cassandra
+    spec:
+      containers:
+      - name: cassandra
+        image: gcr.io/google-samples/cassandra:v11
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 7000
+          name: intra-node
+        - containerPort: 7001
+          name: tls-intra-node
+        - containerPort: 7199
+          name: jmx
+        - containerPort: 9042
+          name: cql
+        resources:
+          limits:
+            cpu: "1"
+            memory: 1Gi
+          requests:
+           cpu: "1"
+           memory: 1Gi
+        securityContext:
+          privileged: true
```

Comment:
@chrislovecnm what is  the reasom for the "priviledged" security setting?
any chance to consider something more specific:
```
          securityContext:
            capabilities:
              add:
                - NET_ADMIN
```